src/BizHawk.Emulation.Cores/Consoles/Sega/SMS/SMS.cs

```diff
@@ -19,9 +19,9 @@ namespace BizHawk.Emulation.Cores.Sega.MasterSystem
 	public partial class SMS : IEmulator, ISoundProvider, ISaveRam, IInputPollable, IRegionable,
 		IDebuggable, ISettable<SMS.SmsSettings, SMS.SmsSyncSettings>, ICodeDataLogger
 	{
-		[CoreConstructor(VSystemID.Raw.SMS)]
-		[CoreConstructor(VSystemID.Raw.SG)]
-		[CoreConstructor(VSystemID.Raw.GG)]
+		[CoreConstructor(VSystemID.Raw.SMS, Priority = CorePriority.Low)]
+		[CoreConstructor(VSystemID.Raw.SG, Priority = CorePriority.Low)]
+		[CoreConstructor(VSystemID.Raw.GG, Priority = CorePriority.Low)]

```

Comment:
I don't think this is necessary? I've forgotten when exactly `Priority` has an effect, but it's only in some edge case or other.

Reply:
Priority is only intended to ever interact with fallback logic. If only two cores are present, it is meaningless.

Reply:
(read: this is not necessary and can be reverted)